packages/node/test/fixtures/config/config.ts

```diff
@@ -12,13 +12,51 @@ export function buildTrigger(overrides?: Partial<RequestTrigger>): RequestTrigge
 }
 
 export function buildConfig(overrides?: Partial<Config>): Config {
+  const oisTitle = 'currency-converter-ois';
+  const securitySchemeName = 'mySecurityScheme';
+  const securitySchemeEnvName = `ss_${oisTitle}_${securitySchemeName}`;
+  const chainType = 'evm';
+  const chainId = '31337';
+  const chainProviderName = 'evm-local';
+  const chainProviderEnvName = `cp_${chainType}_${chainId}_${chainProviderName}`;
+  process.env[securitySchemeEnvName] = 'supersecret';
```

Comment:
I don't think we should be setting environment variables (or any side effects) from fixtures. The job of these functions is only to build and return an object with a specific structure & type. 

These should either be set in the test, a `beforeEach` block or in `test/setup/init/set-env-vars.ts`. (I don't like the nested structure of the `test/setup/` directory and want to change it)

Reply:
Agreed. I think setting them all in `test/setup/init/set-env-vars.ts` is good enough here (because of https://github.com/api3dao/airnode/pull/250#discussion_r599534099) and consistent with how the mnemonic is treated.
Came across this btw, but the current implementation is more obvious, to me at least
https://jestjs.io/docs/environment-variables

Reply:
I agree that `test/setup/init/set-env-vars.ts` is good enough now but I think we need to change it to the future to beforeEach(), specially due to concurrency running the tests. Specially if we do this kind of initialisation: 
https://github.com/api3dao/airnode/blob/master/packages/node/src/handlers/call-api.test.ts#L11, but agree this is good enough here.